proposal.md

```diff
@@ -17,19 +17,19 @@ Cheryl Schramm
 * [Timetable](#timetable)
 * [Facilities](#facilities)
 * [Degree Relevance](#degree-relevance)
-	* [Thao-Tran Le-Phuong](#thao-tran-le-phuong)
+	* [Thao-Tran Le-Phuong](#thao-tran-le -phuong)
 	* [Honor Lopes](#honor-lopes)
 	* [Riley MacKinnon](#riley-mackinnon)
 	* [Igor Veselinovic](#igor-veselinovic)
 
 ## Objectives
 The objective of this project is to develop a messaging platform that is formatted as a blank document that allows for 
-real-time collaboration between users.
+real-time collaboration between users. The platform will display conversations in a document format where users will be able to add or delete anything that they, or others, have said. The platform will also have a 'mini-map' to display where other users' cursors are in the conversation.
```

Comment:
Maybe talk about how we want to build upon the current real-time collaboration implementation.